BUILD.bazel

```diff
@@ -1132,6 +1133,7 @@ cc_binary(
         ":register",
         ":stablehlo_passes",
         ":tosa_passes",
+        "//stablehlo/tests:check_ops",
```

Comment:
Does CMakeList in `tools/` need any updates?

Reply:
Weirdly enough, I guess not, since everything builds and runs fine. But I added check ops just in case.